### PR TITLE
Hyperledger Indy 2024 Q3 Quarterly Report

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Comment by swcurran** [comment]
Still to be reviewed by the Indy Maintainers.  I'll remove the "Draft" label when that has been done.

File: project-reports/2024/2024-Q3-Hyperledger-Indy.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+[did:indy Specification]: https://hyperledger.github.io/indy-did-method/
+[did:indy Specification source repository]: https://github.com/hyperledger/indy-did-method
+[did:indy Networks]: https://github.com/hyperledger/indy-did-networks
+[Indy HIPE]: https://github.com/hyperledger/indy-hipe
+[indy-did-method]: https://github.com/hyperledger/indy-did-method
+
+## Project Health
+
+The project less contributor activity over the quarter. Per the [Indy Quarterly
```
Comment (WadeBarnes) [inline project-reports/2024/2024-Q3-Hyperledger-Indy.md]:
```suggestion
The project had less contributor activity over the quarter. Per the [Indy Quarterly
```

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Minor wording update suggestion.

File: project-reports/2024/2024-Q3-Hyperledger-Indy.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+Activity Dashboard], in the months April through June, 2024 there were 426
+commits (down about 50%)) and 77 contributors (down about 30%). Work was focused
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-Indy.md]:
```suggestion
commits (down about 50%) and 77 contributors (down about 30%). Work was focused
```

File: project-reports/2024/2024-Q3-Hyperledger-Indy.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+[Hyperledger Mentorship]: https://wiki.hyperledger.org/display/INTERN/Hyperledger+Mentorship+Program
+[Indy Read Replica]: https://wiki.hyperledger.org/display/INTERN/Project+Plan+-+Indy+Read+Replica+Implementation
+[Bryan Elee]: https://github.com/rxbryan
+
+Two followups meetings to the most recent Indy Ecosystem Summit (#4, held April
+9, 2024) were held in May, 2024 covering the technical aspects and funding
+approaches to implementing Ledger Redactibility on Indy. A design was developed,
+but little progress was made towards a cross-community collaboration for funding
+and implementing the work. A couple of organizations have (as was suggested)
+submitted a proposal to for grant funding for the effort.
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-Indy.md]:
```suggestion
submitted a proposal for grant funding for the effort.
```

**Review by tkuhrt — APPROVED** [review]
Minor cleanup needed, but looks good.

**Review by petermetz — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-Indy.md (hunk 3 above)
Comment (swcurran) [reply project-reports/2024/2024-Q3-Hyperledger-Indy.md]: Fixed

File: project-reports/2024/2024-Q3-Hyperledger-Indy.md (hunk 2 above)
Comment (swcurran) [reply project-reports/2024/2024-Q3-Hyperledger-Indy.md]: Fixed

File: project-reports/2024/2024-Q3-Hyperledger-Indy.md (hunk 1 above)
Comment (swcurran) [reply project-reports/2024/2024-Q3-Hyperledger-Indy.md]: Fixed

**Review by WadeBarnes — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-Indy.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+[did:indy Specification source repository]: https://github.com/hyperledger/indy-did-method
+[did:indy Networks]: https://github.com/hyperledger/indy-did-networks
+[Indy HIPE]: https://github.com/hyperledger/indy-hipe
+[indy-did-method]: https://github.com/hyperledger/indy-did-method
+
+## Project Health
+
+The project had less contributor activity over the quarter. Per the [Indy Quarterly
+Activity Dashboard], in the months April through June, 2024 there were 426
+commits (down about 50%) and 77 contributors (down about 30%). Work was focused
```
Comment (arsulegai) [inline project-reports/2024/2024-Q3-Hyperledger-Indy.md:60]: Q: re: the 30% decrease in contributors, is it due to reduced activity on Indy All less Indy Besu projects or are there other factors to consider??

**Review by arsulegai — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-Indy.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+Work continued this quarter on the Ubuntu 22.04 version of Indy.
+
+The [indy-vdr] work focused on improving performance in Wallets, and especially
+caching immutable transactions.
+
+Progress was made on the [Indy Besu] implementation of Indy capabilities on
+a Besu base, and in defining the Indy Besu updates to the [Indy DID Method].
+
+[Indy Besu]: https://github.com/hyperledger/indy-besu
+[Indy DID Method]: https://github.com/hyperledger/indy-did-method
```
Comment (Toktar) [inline project-reports/2024/2024-Q3-Hyperledger-Indy.md:97]: We can also note that work to implement the revocation feature is underway right now.

**Review by denyeart — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-Indy.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+### Issues from previous reports
+
+#### **Diversity of Contributor Community**
+
+See the updated type of information in the appropriate section of this report.
+
+## Releases
+
+Release since the last quarterly report -- those after September 30, 2023:
```
Comment (jimthematrix) [inline project-reports/2024/2024-Q3-Hyperledger-Indy.md]: the time of the last quarterly report should be updated to avoid confusion

**Review by jimthematrix — APPROVED** [review]
minor updates requested but with approval

File: project-reports/2024/2024-Q3-Hyperledger-Indy.md (hunk 6 above)
Comment (swcurran) [reply project-reports/2024/2024-Q3-Hyperledger-Indy.md]: Thanks — removed the date reference there as not helpful.

**Pull request merged**